ddd-koans/EduTools and basics/Code editor/task.html

```diff
@@ -0,0 +1,20 @@
+<html>
+<h1>Code editor</h1>
+<p>Editor is the middle bar/window available in the Intellij window.</p>
+
+<p>
+    This is the place where you will write your solutions for the incoming challenges.
+    Majority of the tasks are verified using test written in JUnit 5 library and Kotlin.
+    Rest of the task are just not verified, as they have just the informational purpose.
```

Comment:
-> `Rest of tasks are not verified, as they have only the informational purpose.`